src/cogs/jaegeraccounts.py

```diff
@@ -227,7 +227,7 @@ async def account(self, ctx):
 
     @commands.guild_only()
     @account.command()
-    async def book(self, ctx):
+    async def book(self, ctx, *, args):
```

Comment:
You can put in named optional arguments like so: `async def book(self, ctx, duration=DEFAULT):`

Reply:
I'll need to look this up. Not sure if it will require command to be !account book duration=1 or smth

Reply:
It just makes the variable optional with a default value of DEFAULT.
So in case of `!account book` the duration variable would be DEFAULT,
while in case of `!account book abcdef` duration would be abcdef.
d.py has a builtin parser that just assigns abcdef to that variable for you if applicable.

Reply:
done